diff --git a/dist/router/define-router.js b/dist/router/define-router.js
index 19b0e91c23ecee060e979b67afd30b35fe22414a..1033075d00dc381cdd24de495ff01dfb3b1ca1c2 100644
--- a/dist/router/define-router.js
+++ b/dist/router/define-router.js
@@ -735,8 +735,8 @@ export function unstable_defineRouter(fns) {
         const buildElementCache = createElementCache((cacheId, serialized)=>{
             serializedCachedElements.set(cacheId, serialized);
         });
-        // hard-coded concurrency limit
-        const { runTask, waitForTasks } = createTaskRunner(500);
+        // Keep static generation within Vercel’s 8 GB build machine.
+        const { runTask, waitForTasks } = createTaskRunner(8);
         const skipBuild = fns.unstable_skipBuild;
         // static api
         for (const item of configs){
